appium/webdriver/webdriver.py

```diff
@@ -442,10 +444,13 @@ def set_value(self: T, element: MobileWebElement, value: str) -> T:
         Returns:
             `appium.webdriver.webdriver.WebDriver`: Self instance
         """
-        data = {
-            'id': element.id,
-            'value': [value],
-        }
+        warnings.warn(
+            'The "setValue" API is deprecated and will be removed in future versions. '
```

Comment:
we should also mention which endpoint to use instead

Reply:
So would this suffice? Or would something else be better?

```python
        warnings.warn(
            'The "setText" API is deprecated and will be removed in future versions. '
            'Instead please use the "sendKeys` API. '
            'See https://github.com/appium/python-client/pull/831',
            DeprecationWarning,
        )
```

Reply:
the python name for this api is `send_keys`

we can also propose W3C Actions (`ActionChains` Selenium class) as possible alternative

Reply:
Of course, sorry :)

I am fairly new to Appium, so I am not even sure how `ActionChains` would be used for inserting text.

Reply:
https://appiumpro.com/editions/46-sending-arbitrary-keystrokes-with-the-actions-api

Reply:
I've made a proposed change, which names both. Then hopefully someone can go to this PR, and see any examples :)